.github/workflows/release.yaml

```diff
@@ -0,0 +1,74 @@
+name: Build & upload PyPI package
```

Comment:
How is this made dependent on the check workflow to ensure we don't upload unchecked releases?

Reply:
Yeah, that would be better.

We could get the wheels that were uploaded as artifacts in the test job:
https://github.com/sbraz/pymediainfo/pull/146/files#diff-78ef9dc8d6ce0002e26fc4adde9d00123acd7a7c214da456a828814850c7d9e2R151-R157

I am not sure we can download the artifacts from another workflow, then we would have only one workflow (= one yaml file)

Reply:
Is it a big problem to have only one workflow though?